homeassistant/components/reolink/config_flow.py

```diff
@@ -0,0 +1,137 @@
+"""Config flow for the Reolink camera component."""
+from __future__ import annotations
+
+import logging
+from typing import cast
+
+from reolink_ip.exceptions import ApiError, CredentialsInvalidError
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_PORT, CONF_USERNAME
+from homeassistant.core import callback
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers import config_validation as cv
+
+from .const import CONF_PROTOCOL, CONF_USE_HTTPS, DEFAULT_PROTOCOL, DOMAIN
+from .host import ReolinkHost
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ReolinkOptionsFlowHandler(config_entries.OptionsFlow):
+    """Handle Reolink options."""
+
+    def __init__(self, config_entry):
+        """Initialize ReolinkOptionsFlowHandler."""
+        self.config_entry = config_entry
+
+    async def async_step_init(self, user_input=None) -> FlowResult:
```

Comment:
Incomplete typing. Please type the whole signature when adding type annotations.